src/amphtml-email/20_Samples/Form_Inside_List.html

```diff
@@ -0,0 +1,73 @@
+<!---
+
+preview: amp4email
+skipValidation: 'true'
+draft: 'true'
+
+--->
+
+<!--
+## Introduction
+
+This sample demonstrates how to update data inside an [`amp-list`](https://www.ampproject.org/docs/reference/components/amp-list)
+based on a user's subsequent input in an [`amp-form`](https://www.ampproject.org/docs/reference/components/amp-form).
+-->
+
+<!-- -->
+<!doctype html>
+<html ⚡4email>
+<head>
+  <meta charset="utf-8">
+  <script async src="https://cdn.ampproject.org/v0.js"></script>
+  <script async custom-element="amp-list" src="https://cdn.ampproject.org/v0/amp-list-0.1.js"></script>
+  <script async custom-element="amp-form" src="https://cdn.ampproject.org/v0/amp-form-0.1.js"></script>
+  <script async custom-template="amp-mustache" src="https://cdn.ampproject.org/v0/amp-mustache-0.2.js"></script>
+  <style amp4email-boilerplate>body{visibility:hidden}</style>
+  <title>Form Inside List</title>
+</head>
+<!-- ## Implementation -->
+<!-- -->
+<body>
+  <!--
+    Because nested templates are not allowed, define the template
+    that `amp-form` will use and give it an `id`.
+  -->
+  <template id="nickname-template" type="amp-mustache">
+    {{#nickname}}
+    <p>The nickname you chose is {{nickname}}.</p>
+    {{/nickname}}
+    {{^nickname}}
+    <p>No nickname selected.</p>
+    {{/nickname}}
+  </template>
+
+  <!--
+    Then define the template that the list will use, which will contain an `amp-form`
```

Comment:
replace "Then define the template that the list will use" with "Then define the template that `amp-list` will use"